packages/frontend/src/functions/File.ts

```diff
@@ -25,10 +25,17 @@ export default class File {
   get model() {
     return this._model;
   }
+  private editor: editorType.ICodeEditor;
 
   private applyingRemote = false;
 
-  constructor(path: string, content: RGAJSON, model: editorType.ITextModel) {
+  constructor(
+    path: string,
+    content: RGAJSON,
+    model: editorType.ITextModel,
+    editor: editorType.ICodeEditor
```

Comment:
We never use  "editor". Any reason why pass it in?

Reply:
Nice catch! That was done for testing purpouses earlier!